### PR TITLE
New settings for image mirroring and rotation, added "Not Authenticated" state as option from query results

### PR DESCRIPTION
**Pull request by max-at-groundlight**
You can now specify whether an image is flipped or rotated from configuration website. The default is rotated 180 degrees and un flipped, the default settings to work on the M5Stack Timer Camera series.

### CODE REVIEW

File: src/deployable_example.cpp [hunk 1]
```diff
@@ -78,6 +79,8 @@ String queryStateToString (QueryState state) {
       return "LAST_RESPONSE_FAIL";
     case LAST_RESPONSE_UNSURE:
       return "LAST_RESPONSE_UNSURE";
+    case NOT_AUTHENTICATED:
+      return "NOT_AUTHENTICATED";
```
Comment (positavi) [inline src/deployable_example.cpp:83]: this part is not about mirroring in rotation.  please update the PR title to include the functionality.

File: src/deployable_example.cpp [hunk 2]
```diff
@@ -950,12 +954,33 @@ bool try_save_config(char * input) {
     } else {
       preferences.remove("motion");
     }
+    if (doc["additional_config"].containsKey("flip_vert")) {
+      debug("Has flip vert!");
```
Comment (positavi) [inline src/deployable_example.cpp]: could be a more descriptive debug message.

**Review by positavi — DISMISSED** [review]
comments inline.  also please update PR title to include description of non-camera changes.

File: src/deployable_example.cpp (hunk 1 above)
Comment (max-at-groundlight) [reply src/deployable_example.cpp:83]: PR title updated

File: src/deployable_example.cpp (hunk 2 above)
Comment (max-at-groundlight) [reply src/deployable_example.cpp]: fixed

**Review by positavi — COMMENTED** [review]
ok to merge after you fix the strings as discussed.

**Pull request merged**